website/docs/r/org_vdc.html.markdown

```diff
@@ -107,9 +100,6 @@ The following arguments are supported:
 
 Capacity must be specified twice, once for `memory` and another for `cpu`.  Each has the same structure:
 
-* `units` - (Required) Units in which capacity is allocated. For CPU capacity, one of: {MHz, GHz}.  For memory capacity, one of: {MB, GB}.
-* `allocated` - (Optional) Capacity that is committed to be available.
-* `limit` - (Optional) Capacity limit relative to the value specified for Allocation. It must not be less than that value. If it is greater than that value, it implies overprovisioning.
-* `reserved` - (Optional) Capacity reserved
-* `used` - (Optional) Capacity used. If the VDC AllocationModel is ReservationPool, this number represents the percentage of the reservation that is in use. For all other allocation models, it represents the percentage of the allocation that is in use.
-* `overhead` - (Optional) Number of Units allocated to system resources such as vShield Manager virtual machines and shadow virtual machines provisioned from this Provider VDC.
\ No newline at end of file
+* `allocated` - (Required) Capacity that is committed to be available. Value in MB or MHz.
```

Comment:
We need this "Value in MB or MHz." in some other parameters too I believe (limit, reserved).

Reply:
Added